docs/user/tutorials/01-30-set-up-mtls-gateway.md

```diff
@@ -43,12 +37,42 @@ The procedure of setting up a working mTLS Gateway is described in the following
 
 3. Set up Istio Gateway in mutual mode.
 
-    Assuming that you have successfully created the server certificate and it is stored in the `kyma-mtls-certs` Secret within the default namespace, modify and apply the following Gateway custom resource in a cluster:
+    <!-- tabs:start -->
+    #### **Kyma Dashboard**
+    1. Go to **Istio > Gateways** and select **Create**.
+    2. Provide the following configuration details:
+      - **Name**: `kyma-mtls-gateway`
+      - Add the selectors:
+          - **app**: `istio-ingressgateway`
```

Comment:
We didn't add the app selector in the previous tutorial `01-20-set-up-tls-gateway.md`. Do we also want to have it there? Also istio selector is mentioned here, when it is already there by default.

Reply:
Removed